common/protob/messages-management.proto

```diff
@@ -36,6 +36,15 @@ enum HomescreenFormat {
     Jpeg240x240 = 2;
 }
 
+
+/**
+ * Device color
+ */
+enum DeviceColor {
```

Comment:
**device** is one of these words we use so much it's sometimes hard to tell what it means. What about `VariantColor`? Or `ChassisColor`? Though that wouldn't work with btconly, if I understand it correctly that we want to be able to enforce btconly firmware based on OTP contents.

Reply:
Well the enum itself is gone as part of https://github.com/trezor/trezor-firmware/pull/2961/commits/adb743ef4444e9a2ff05e8a73928ad70173a9f6a , but the term `device` is still used everywhere else in this feature - and i think its rightful usage in this case, as the data/variants are bound to the specific device.

Though there is i.e. `device_id` which is bound to the storage instance and gets regenerated on wipe, so yeah, there is room for confusion.

One alternative term i can think of is 'unit', so that would result in unit_color and unit_btconly, not sure if that is better. 

Or i can drop the device word and just call the whole thing variant, but thats kinda opaque.

Any preference?

Reply:
_Unit_'s good, but I can live with _device_ also. We can discuss on standup but I'd leave the final choice to you.

Reply:
going with unit: https://github.com/trezor/trezor-firmware/commit/4b3267b66829098c17fd6416832ebf746e62413b